<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit</class>
 <widget class="QDialog" name="deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="input_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>391</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #E2E4E6;
color: black;

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Deposit calculator</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>143</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>356</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #000;
border-bottom: 1px solid #b5b5b5;
border-radius:  5px;
background-color: #fff;
}

QPushButton::pressed {
background-color: grey;
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>149</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>66</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Deposit amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deposit_amount_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>146</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Interest rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interest_rate_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>186</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Tax rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tax_rate_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>216</y>
      <width>331</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Periodicity of payments</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="monthly_btn">
           <property name="text">
            <string>monthly</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="yearly_btn">
           <property name="text">
            <string>yearly</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="is_capital">
       <property name="text">
        <string>capitalization of interest</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>276</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Monthly replenishment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replenish_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>306</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Monthly withdrawal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="withdraw_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>106</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Term</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="term_val">
         <property name="styleSheet">
          <string notr="true">background-color: white;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QRadioButton" name="months">
         <property name="text">
          <string>months</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="years">
         <property name="text">
          <string>years</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="output_frame">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>-10</y>
     <width>201</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F7CFCA;
color: black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="earned">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Total earned</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="earned_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="tax">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Tax amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="taxed_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="total">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Total amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
qproperty-alignment: AlignCenter;
}</string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
